code-challenges.test.js

```diff
@@ -14,20 +14,60 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("fibonacci", () => {
+  it("returns an array that length containing the numbers of the Fibonacci sequence.", () => {
+    expect(fibonacci(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+    expect(fibonacci(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+  })
+})
+
+
 const fibLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+
+
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
 
-// --------------------2) Create a function that takes in an object and returns an array of the values sorted from least to greatest.
-// Hint: Check out this resource: Object.values() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values
+// 1.) Declare a fuction to set up the fibonacci sequence. Create a function that has an array with 0 and 1.
 
-// a) Create a test with expect statements for each of the variables provided.
+function fibonacci(n) {
+    const fib = [1, 1]
+
+// 2.) Make a for loop to iterate on the array.
+  for (let i = 0; i< n - 2; i++) {
+
+    // 3.) Have fib identify the index in the array and have it equal to the previous 2 indexes.
+    fib[i + 2] = fib[i + 0] + fib[i + 1]
+  }
+  // 4.) Have it return the new array if the fibonacci sequence
+    return fib
+}
+
+// console.log(fibonacci(fibLength1))
+// console.log(fibonacci(fibLength2))
+
+
+// // // --------------------2) Create a function that takes in an object and returns an array of the values sorted from least to greatest.
+// // // Hint: Check out this resource: Object.values() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values
+
+// // // // a) Create a test with expect statements for each of the variables provided.
+
+describe("leastToGreatest", () => {
+  it("Create a test with expect statements for each of the variables provided.", () => {
+      expect(dayArray(studyMinutesWeek1).toEqual([15, 15, 20, 30, 30, 60, 90]))
+      expect(dayArray(studyMinutesWeek2).toEqual([10, 15, 20, 45, 60, 65, 100]))
+  })
+})
+
+// TypeError: dayArray(...).toEqual is not a function
+
+// It's giving me this error code and i am unable to detect the problem. To my knowledge the function i wrote works and i feel like I also wrote the test right so im confused on what i did wrong.
```

Comment:
At the end of lines 63 and 64, one of those closing parentheses should be before the `.toEqual`, to close the parentheses for `expect()`